sys-fs/bees/bees-9999.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit git-r3 eutils multilib
```

Comment:
You don't need `eutils` or `multilib`

`git-r3` should be moved to inside the `if [[ ${PV} == "9999" ]] ; the ` conditional, after `EGIT_BRANCH` is set

Reply:
Oops, yes, eutils is a leftover from a previous version. And I didn't expect you can to multiple inherit calls.